dts/bindings/gpio/ti,tca9538.yaml

```diff
@@ -24,6 +24,21 @@ properties:
       Connection for the NINT signal. This signal is active-low when
       produced by tca9538 GPIO node.
 
+  input-latch:
+    type: int
+    description: |
+      Input latch register bit is 0 by default and the input pin state
+      is not latched. When input latch register bit is 1 and the input
+      pin state is latched.
+
+  interrupt-mask:
+    type: int
+    description: |
+      Interrupt mask register is set to logic 1 by default without
+      enabling interrupts. Setting corresponding mask bits to logic
+      0 to enable the interrupts.
+
+
```

Comment:
nits: unnecessary empty line

Reply:
Thank you for the comment, will do it later if there's any comments requires code changes

Reply:
still there, sorry don't mean to be harsh on nitpicks but if we start not caring the code base is going to become a giant mess, please fix

Reply:
Yeah, has been updated!